tests/__mocks__/handlers.ts

```
@@ -70,7 +70,9 @@ import pull101CommentsGet from "./routes/pull-101-work-ubq-fi/pull-101-comments-
 import pull101Get from "./routes/pull-101-work-ubq-fi/pull-101-get.json";
 import pull101ReviewsGet from "./routes/pull-101-work-ubq-fi/pull-101-reviews-get.json";
 import issue12CommentsGet from "./routes/pull-12-conversation-rewards/issue-12-comments-get.json";
+import issue12TimelineGet from "./routes/pull-12-conversation-rewards/issue-12-timeline-get.json";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'issue12TimelineGet'. Consider to delete it or rename it.

'issue12TimelineGet' is defined here:


(lint/suspicious/noRedeclare)